doc/MANUAL.adoc

```diff
@@ -929,6 +930,33 @@ Optional attributes:
 * *update-mtime*: If *true*, update the modification time (mtime) of cache
   entries that are read. The default is *false*.
 
+=== Redis storage backend
+
+URL format: `redis://HOST:PORT` or `redis://UNIX_SOCK`
+
+This backend stores data in a Redis (or Redis-compatible) server.
+There are implementations for both memory and disk based storage.
+
+See <https://redis.io>
+
+Note that ccache will not perform any cleanup of the Redis storage.
+But you can configure LRU eviction: <https://redis.io/topics/lru-cache>
+
+In addition to the in-memory database, there are also persistent NoSQL.
+See <https://github.com/yinqiwen/ardb> (using https://rocksdb.org/[RocksDB] or similar)
```

Comment:
This is great information, but I feel that it's a bit too specific and volatile to be part of the documentation. When you read the ccache 4.4 man page three years from now it's maybe some other Redis-compatible software that is the one to prefer, URLs may have changed, etc.

One idea I had when sketching the configuration format was to put this kind of information on the ccache wiki instead: 
https://github.com/ccache/ccache/commit/ed1f6622123315977e95e44a1ecb39202852b88d#diff-ecf215ac3df4ba49cf18dc85a8c137f0567539bd19e17270063d59e15f412667R926-R927

What do you think, does that make sense?

Reply:
I will keep the reference to "LRU eviction" (should be easy enough to find, even if documentation is rearranged)